modelname: "../rsc/atlas/robot.urdf"
activation_key: "../share/raisim/activation.raisim"
enable_gravity: True
animation_mode: True
jointPgain: 800.0
jointDgain: 10
